apps/docs/src/routes/architecture/provider-service/+page.md

```
@@ -0,0 +1,313 @@
+}
+```
+
+Additional claims are **optional**.
+
+---
+
+## 6. Migration Support
+
+### 6.1 Export endpoint
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)